.github/workflows/releaseFail.yml

```diff
@@ -0,0 +1,44 @@
+# test
+on:
+  workflow_run:
+    workflows:
+      - RC
+      - promote-rc-to-latest
+      - manual promote
+      - pr for rc
+      - rc tag/release
+    types:
+      - completed
+
+jobs:
+  failure-notify:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'failure' }}
+    steps:
+      - name: Announce Failure
+        id: slack
+        uses: slackapi/slack-github-action@v1.21.0
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.CLI_ALERTS_SLACK_WEBHOOK }}
+          SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+        with:
+          payload: |
+            {
+              "text": "Release failed: ${{ github.event.workflow_run.repository.name }}",
```

Comment:
I am not super familiar with Slack's Block syntax. Is this `text` that is outside of the `blocks` array correct?

Reply:
it's the fallback text (ex: notificaiton on your phone where the richer stuff isn't supported)

https://api.slack.com/reference/messaging/payload

Reply:
Good to know, thanks 👍